test/Grains/BenchmarkGrains/GrainStorage/PersistentGrain.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using Orleans;
+using BenchmarkGrainInterfaces.GrainStorage;
+using Microsoft.Extensions.Logging;
+
+namespace BenchmarkGrains.GrainStorage
+{
+    public class PersistentGrain : Grain<int>, IPersistentGrain
+    {
+        private readonly ILogger<PersistentGrain> logger;
+
+        public PersistentGrain(ILogger<PersistentGrain> logger)
+        {
+            this.logger = logger;
+        }
+
+        public async Task<int> Set(int value)
+        {
+            Stopwatch sw = Stopwatch.StartNew();
+            try
+            {
+                this.State = value;
+                await this.WriteStateAsync();
+                sw.Stop();
+                object[] args = { this.GetPrimaryKey(), sw.ElapsedMilliseconds };
```

Comment:
I don't quite understand the reasoning behind an explicit array, but that's fine.

Reply:
Stupid heap analyzer was complaining about everything.. I turned it off :/  This code was written while it was still on and being neurotic.

Reply:
hahaha, yeah, my bad. It is definitely too picky. At most it should just give you friendly hints, never warnings.

Reply:
Instead, maybe we can add it to the solution with a rules file which reduces the severity of all of its diagnostics to the lowest visible level (which doesn't show up in build output - if that exists). Obviously you need to allocate some times and sometimes you simply don't care (since it's not performance critical code).

For the test subtree, we should have it disabled.